rb/spec/integration/selenium/webdriver/fedcm_spec.rb

```diff
@@ -0,0 +1,86 @@
+require_relative 'spec_helper'
+
+module Selenium
+  module WebDriver
+    module FedCM
+      describe FedCM, only: { browser: :chrome }, exclusive: {bidi: false, reason: 'Not yet implemented with BiDi'} do
+        let(:dialog) { driver.fedcm_dialog }
+
+        before do
+          driver.get url_for('fedcm/fedcm.html')
+        end
+
+        context 'without dialog present' do
+          it 'throws an error when getting the title' do
```

Comment:
Because the integration tests are expensive and require execution of all hooks/restart a browser in case of failure, we prefer to consolidate tests whenever possible. For example, in this case, all raise error tests can be joined into a single test:

```ruby
context 'without dialog present' do
  it 'throws an error' do
    expect { dialog.title }.to raise_error(Error::NoSuchAlertError)
    expect { dialog.subtitle }.to raise_error(Error::NoSuchAlertError)
    ...
```

The rest of the integration can be joined as well around their behavior.